source/h2_stream.c

```diff
@@ -22,6 +22,10 @@ static int s_stream_get_sent_error_code(struct aws_http_stream *stream_base, uin
 
 static void s_stream_cross_thread_work_task(struct aws_channel_task *task, void *arg, enum aws_task_status status);
 static struct aws_h2err s_send_rst_and_close_stream(struct aws_h2_stream *stream, struct aws_h2err stream_error);
+static int s_stream_reset_stream_internal(
+    struct aws_http_stream *stream_base,
+    uint32_t http2_error,
+    int aws_error_code);
```

Comment:
would it look nicer here to pass `struct aws_h2err stream_error` instead of two separate error codes: `uint32_t http2_error, int aws_error_code`?

Reply:
Totally forgot we have `struct aws_h2err`